meshnets/utils/model_training.py

```diff
@@ -7,18 +7,14 @@
 from pytorch_lightning.callbacks import ModelCheckpoint
 import torch
 
+import datasets
+
 import ray
 import ray.train.lightning
 
-from meshnets.modules.lightning_wrapper import MGNLightningWrapper
-from meshnets.modules.model import MeshGraphNet
-from meshnets.utils.callbacks import GeometricBatchSize
-from meshnets.utils.callbacks import GPUUsage
-from meshnets.utils.callbacks import GradientNorm
-from meshnets.utils.callbacks import MLFlowLoggerFinalizeCheckpointer
-
-import meshnets
-import datasets
+from .. import modules
```

Comment:
I had it like this before. The problem is that `model_training.py` is part of the meshnets package:

```
meshnets
├── __init__.py
├── data_processing
│   ├── data_mappers.py
│   ├── __init__.py
│   ├── torch_utils.py
├── modules
│   ├── __init__.py
│   ├── lightning_wrapper.py
│   ├── model.py
└── utils
    ├── callbacks.py
    ├── __init__.py
    ├── model_training.py
```

Therefore importing meshnets was raising the `circular imports` warning. I don't think we should import `meshnets` inside `meshnets`. But please let me know if there is a better way to go about this

Reply:
Yes, but you are still techinically doing the same with your relative imports. In either case, notice that now you aren't importing meshnets with `import meshnets` or `import .` that might have been the root cause. Otherwise, you need to structure the inits to not have the circular imports, this also tells you that we may have problems with the structure of the code.

Reply:
I will take a look into this and will report back

Reply:
Removed the relative imports